hyp3_autorift/vend/testautoRIFT.py

```diff
@@ -286,6 +286,8 @@ def runAutorift(I1, I2, xGrid, yGrid, Dx0, Dy0, SRx0, SRy0, CSMINx0, CSMINy0, CS
         obj.preprocess_filt_wal_nodata_fill()
     elif 'wallis' in preprocessing_methods:
         obj.preprocess_filt_wal()
+    elif 'fft' in preprocessing_methods:
+        print('fft filtering must be done before processing with geogrid! Be careful when using this method')
```

Comment:
```suggestion
        warnings.warn('fft filtering must be done before processing with geogrid! Be careful when using this method', UserWarning)
        # obj.preprocess_filt_wal()
        # obj.preprocess_filt_fft()
```
It might be worth switching this to a warning and leaving the "autoRIFT" expected calls here as a comment so you can trace what the FFT filter is from this point in the code.

Reply:
Made this change!